db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
```

Comment:
Well, its running on my side. Below are the screenshots
![Capture1](https://user-images.githubusercontent.com/47685927/223658376-a54d511c-ddb9-4aed-aede-7b9eb5ba7134.PNG)
![Capture2](https://user-images.githubusercontent.com/47685927/223658224-5ba2d4c2-5c55-42c0-a388-18ee76bba7b0.PNG)
![Capture3](https://user-images.githubusercontent.com/47685927/223658260-1df29019-786d-4833-8be7-8b4f8ec30852.PNG)

Reply:
My rails server runs perfectly

Reply:
And the api also has a response as shown below
![Capture4](https://user-images.githubusercontent.com/47685927/223659301-0f66bd64-7f5d-4f98-97a2-1328306c67f1.PNG)

Reply:
I think you didn't read my comment to the end.
I told you this problem happened because you didn't add greetings seeds to your app.
Next time please read the whole comment.